pulsar-broker/src/main/java/org/apache/pulsar/broker/service/persistent/PersistentTopic.java

```diff
@@ -2107,8 +2108,7 @@ public void updateRates(NamespaceStats nsStats, NamespaceBundleStats bundleStats
 
         replicators.forEach((region, replicator) -> replicator.updateRates());
 
-        nsStats.producerCount += producers.size();
-        bundleStats.producerCount += producers.size();
+        final AtomicInteger producerCount = new AtomicInteger();
```

Comment:
I guess this could be a `org.apache.commons.lang3.mutable.MutableInt` since this is used in single threaded code.

Reply:
Replaced `AtomicInteger` with `MutableInt`.